submissions/onyinyeChukwuka/css/sign-up.css

```diff
@@ -0,0 +1,290 @@
+body {
+    font-family: 'Montserrat', sans-serif;
+    background-color: #ffffff;
+}
+
+.header {
+    justify-content: space-between;
+    padding: 2px;
+    background-color: #0066f5;
+}
+
+nav {
+    padding: 20px;
+}
+
+.left-side {
+    display: flex;
+}
+
+nav>.right-side>div {
+    margin-right: 100px;
+    font-size: 16px;
+    color: #ffffff;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.19;
+    letter-spacing: normal;
+    text-align: left;
+}
+
+.tab {
+    color: #ffffff;
+    text-decoration: none;
+    font-weight: normal;
+}
+
+.link-wrapper {
+    height: 22px;
+}
+
+.link-wrapper a {
+    color: #ffffff;
+    text-decoration: none;
+    font-weight: normal;
+}
+
+.sidebar h4 {
+    font-family: Montserrat;
+    font-size: 50px;
+    font-weight: bold;
+    font-style: normal;
```

Comment:
font-style is declared twice @Onyinye101 . By default, font-style and font-weight is normal. I think you should remove such declarations from your css.